packages/react-core/src/components/NotificationDrawer/examples/NotificationDrawer.md

```diff
@@ -11,45 +22,74 @@ import { Button, Dropdown, DropdownItem, DropdownPosition, DropdownDirection, Dr
 import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon';
 
 ## Examples
+
 ```js title=Basic
 import React from 'react';
-import { Dropdown, DropdownItem, DropdownPosition, DropdownDirection, DropdownSeparator, KebabToggle, NotificationDrawer, NotificationDrawerBody, NotificationDrawerHeader, NotificationDrawerList, NotificationDrawerListItem, NotificationDrawerListItemBody, NotificationDrawerListItemHeader, Title } from '@patternfly/react-core';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownPosition,
+  DropdownDirection,
+  DropdownSeparator,
+  KebabToggle,
+  NotificationDrawer,
+  NotificationDrawerBody,
+  NotificationDrawerHeader,
+  NotificationDrawerList,
+  NotificationDrawerListItem,
+  NotificationDrawerListItemBody,
+  NotificationDrawerListItemHeader,
+  Title
+} from '@patternfly/react-core';
 
 class BasicNotificationDrawer extends React.Component {
-constructor(props) {
+  constructor(props) {
     super(props);
     this.state = {
-      isOpen: false,
-      isActive: ''
```

Comment:
Thanks for your suggestions. I think use id is a good way to simplify the whole things so I change the usage in Notification Drawer Groups example (both in the doc and in the demo app) because it has 9 toggle buttons. Here is the way I changed it.

1. Instead of using 9 `onToggle` functions and 9 `isOpen` state, I use only one `isToggle` function and `isOpen` state.
2. `isOpen` state is an object, the key of this object is the id of the ToggleButton and the value of this object is the state of the dropdown menu linked to this toggle button.
3. `onToggle` function receives 2 parameters, the first is the key and the second is the value. When it is toggled, I will set the key and value into `isOpen` state.
4. Only when `isOpen` is not null and `isOpen[id]` is true, this dropdown menu will be expanded.

After changing the usage, the example reduces about 40 lines of code, which looks much better than before. I only change the usage in Notification Drawer Groups example because it uses the most toggle buttons than other examples. I hope that our users could choose what usage they would use depending on the number of toggle buttons.